tests/baselines/reference/assignmentLHSIsValue.errors.txt

```diff
@@ -104,6 +105,8 @@ tests/cases/conformance/expressions/assignmentOperator/assignmentLHSIsValue.ts(7
     
     // object literals
     { a: 0} = value;
+      ~
+!!! error TS7028: Unused label.

```

Comment:
This test should probably be fixed up.  You could wrap the object literal in parens.